docs/rst/user_manual/participants/wan.rst

```diff
@@ -0,0 +1,88 @@
+.. include:: ../../exports/alias.include
+
+.. _user_manual_participants_wan:
+
+###
+WAN
+###
+
+This type of :term:`Participant` refers to a :term:`Discovery Server` :term:`DomainParticipant` that communicates
+with other **WAN** Participants in different networks.
+This Participant will work as bridge for every Participant working locally in the LAN and any other LAN that has
+a |ddsrouter| with an active WAN Participant.
+
+.. warning::
+
+    Do not try to communicate a WAN Participant with any other kind of Participant that is not of type
+    WAN Participant.
+
+Use case
+========
+
+Use this Participant in order to communicate an internal DDS network with other networks in different LANs.
+Each of the networks to communicate will need a running |ddsrouter|, and the messages will be relay from one to
+another depending on the topics filtered by each of them.
+
+
+Type aliases
+============
+
+* ``wan``
+* ``router``
+
+
+Configuration
+=============
+
+WAN Discovery Server Participant allow configure the standard attributes of a Discovery Server.
+To configure the Discovery Server :term:`GuidPrefix`, check the following section
+:ref:`Configuration section <user_manual_configuration_domain_id>`.
+To configure the Discovery Server listening addresses, check the following section
+:ref:`Configuration section <user_manual_configuration_discovery_server_listening_addresses>`.
+To configure the Discovery Server connection addresses to connect with other Discovery Servers,
+check the following section
+:ref:`Configuration section <user_manual_configuration_discovery_server_connection_addresses>`.
+
+.. note::
+
+    Those addresses in *listening-addresses* and *connection-addresses* use ``TCP`` transport by default
+    if the transport is not specified in the address configuration.
+
+Configuration Example
+=====================
+
+Configure a WAN Discovery Server with ROS2 GuidPrefix with id ``2`` (``44.53.02.5f.45.50.52.4f.53.49.4d.41``).
```

Comment:
```suggestion
Configure a Local Discovery Server setting the GuidPrefix used for ROS 2 deployments with id ``2`` (``44.53.02.5f.45.50.52.4f.53.49.4d.41``).
```

Reply:
:-1:  It was my error in the first place.